website/scripts/update-graphql-doc.sh

```diff
@@ -0,0 +1,3 @@
+cd website
```

Comment:
add shebang and `set -e`

Reply:
462dd558c674242483abd8f8ef865aafe2576ed7